src/dot-components/Cart/cart.jsx

```diff
@@ -0,0 +1,85 @@
+
+import "./cart.css"
+import "../../public-css/root.css";
+import { Link } from "react-router-dom";
+import { useCart } from "../../Context/cartContext"
+import { useWishlist } from "../../Context/wishlistContext";
+import { Navbar } from "../Navbar/navbar";
+
+const Cart = () => {
+
+    const { cartState, cartDispatch } = useCart();
+    const { wishState, wishDispatch } = useWishlist();
+
+    const totalPrice = cartState.cart.reduce((acc, curr) => acc + Number(curr.price) * Number(curr.quantity), 0 );
+    const discountPrice = (10*totalPrice)/100;
+    const finalPrice = totalPrice-discountPrice;
+
+    return (
+      <div className="App">
+        <Navbar/>
+    <div class="cart-container margin-30">
```

Comment:
in react, class is written as className
Suggestion to use some plugin to convert them